MetadataDevelopment/ImplementationChecks/implementationCheckEBML_micro.xsl

```diff
@@ -418,6 +419,12 @@
                         <xsl:with-param name="x" select="mmt:MicroMediaTrace/mmt:b[@n='Segment']/mmt:b[@n='Cluster']/mmt:b[@n='SimpleBlock']/mmt:b[@parser='FFV1']/mmt:d[@n='colorspace_type']"/>
                         <xsl:with-param name="list">0 1</xsl:with-param>
                       </xsl:call-template>
+                      <xsl:call-template name="data_is_in_list">
+                        <xsl:with-param name="icid">FFV1-VALID-VERSION</xsl:with-param>
+                        <xsl:with-param name="version">1</xsl:with-param>
+                        <xsl:with-param name="x" select="$ffv1version"/>
+                        <xsl:with-param name="list">0 1 3.4</xsl:with-param>
```

Comment:
From my point of view, our conformance checker should not flag a 3.5 version as invalid, the current version of our conformance checker could be used in the future, when a 3.5 version is out, it would be wrong that the stream is flagged as invalid (future-proof test).